packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -176,7 +176,8 @@ class GestureDetector extends StatelessWidget {
     this.onScaleUpdate,
     this.onScaleEnd,
     this.behavior,
-    this.excludeFromSemantics = false
+    this.excludeFromSemantics = false,
+    this.dragStartBehavior = DragStartBehavior.start,
```

Comment:
How was the default chosen? It feels like this PR contains many, many places where the behavior is changed...

Reply:
Yah, Ian felt 'start' made sense because animations would be smoother. What do you think would be the best way to choose?

Reply:
That sounds like a good reason to chose start as the default. I am cool with that.